templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Manufacturer List
+    </h1>
+  <h1>Manufacturer List
+    <a style="float: right" href="{% url "taxi:manufacturer-create" %}">+</a>
+  </h1>
+
+    {% if manufacturer_list %}
+      <table class="table">
+  {% if manufacturer_list %}
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
+
+      {% for manufacturer in manufacturer_list %}
+        <tr>
+          <th>ID</th>
+          <th>Name</th>
+          <th>Country</th>
```

Comment:
There are duplicate `<th>` elements inside the table row. These should be removed to avoid redundancy and maintain a correct table structure.